apps/website/src/app/learn/page.tsx

```diff
@@ -95,27 +95,32 @@ export default function Learn() {
         {
             title: "Semaphore identities",
             description:
-                "Given to all Semaphore group members, it is comprised of three parts - identity commitment, trapdoor, and nullifier.",
+                "A Semaphore identity is an EdDSA key-pair plus the commitment (i.e. the hash of the public key), which is used as the public value of the Semaphore group members.",
             linkText: "Create Semaphore identities",
             linkUrl: "https://docs.semaphore.pse.dev/guides/identities",
             codeText: `import { Identity } from "@semaphore-protocol/identity"
 
-const { trapdoor, nullifier, commitment } = new Identity()`,
+// Random identity.
+const identity1 = new Identity()
+
+// Passing a secret.
+const identity2 = new Identity("secret")
+`,
             itemList: [
                 {
                     icon: <IconEyelash w="24px" h="24px" color="primary.600" />,
                     heading: "Private values",
```

Comment:
```suggestion
                    heading: "Private value",
```

We will only explain one private value, right?